CMakeLists.txt

```diff
@@ -17,27 +17,28 @@ set(LXQTBT_MINIMUM_VERSION "0.6.0")
 set(QT_MINIMUM_VERSION "5.7.1")
 set(QTXDG_MINIMUM_VERSION "3.3.0")
 
+option(BUILD_WITH_FMQT "Depend on libfm-qt to provied the file dialog" ON)
```

Comment:
I don't agree with that being optional.
As I said before, we are allowing a non coherent user experience.

Reply:
it is in fact optional - we can drop the option and require it. Only in that case it isn't optional.

Reply:
@agaida That's what I said/meant. It should not be optional. It should be required.

Reply:
State of art right now - if libfm.so.X or libfm.so isn't found, the Qt dialog will be used - libfm.so is a development file, so if we don't force packagers do do their work right the result of our work will not be predictable. (it might or might not work as expected)